codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -276,22 +277,25 @@ public class <%= schema_name %> {
                 <% end %>
 
                 <% type.required_input_fields.each do |field| %>
+                  <%= java_annotations(field) -%>
                   public <%= java_input_type(field.type) %> get<%= field.classify_name %>() {
                       return <%= escape_reserved_word(field.camelize_name) %>;
                   }
 
-                  public <%= type.name %> set<%= field.classify_name %>(<%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
+                  public <%= type.name %> set<%= field.classify_name %>(<%= java_annotations(field).gsub(/(.)$/, '\1 ') %><%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
```

Comment:
The `.gsub(/(.)$/, '\1 ')` replacement is really weird, since it just adds a trailing whitespace at the end of each line in the string.

```
irb(main)> "one\ntwo\n".gsub(/(.)$/, '\1 ')
=> "one \ntwo \n"
```

but it looks like what you actually want is to replace the newlines.  You could use `.tr("\n", " ")` instead to do that.

However, why not just add a keyword argument to`java_annotations` to control the separator so you don't need to workaround its limitation with string replacements.